docs/local-server.md

```diff
@@ -99,27 +100,39 @@ With the path "/config" you can get the current configuration.
   "displayBrightness": 100,
   "offlineMode": false,
   "model": "I-9PSL",
-  "monitorDisplayCompensatedValues": true
+  "monitorDisplayCompensatedValues": true,
+  "corrections": {
+    "pm02": {
+      "correctionAlgorithm": "epa_2021",
+      "slr": {}
+      }
+    }
+  }
 }
 ```
 
 #### Set Configuration Parameters (PUT)
 
-Configuration parameters can be changed with a put request to the monitor, e.g.
+Configuration parameters can be changed with a PUT request to the monitor, e.g.
 
 Example to force CO2 calibration
 
- ```curl -X PUT -H "Content-Type: application/json" -d '{"co2CalibrationRequested":true}' http://airgradient_84fce612eff4.local/config ```
+ ```bash
+ curl -X PUT -H "Content-Type: application/json" -d '{"co2CalibrationRequested":true}' http://airgradient_84fce612eff4.local/config 
+ ```
 
 Example to set monitor to Celsius
 
- ```curl -X PUT -H "Content-Type: application/json" -d '{"temperatureUnit":"c"}' http://airgradient_84fce612eff4.local/config ```
+ ```bash
+ curl -X PUT -H "Content-Type: application/json" -d '{"temperatureUnit":"c"}' http://airgradient_84fce612eff4.local/config 
+ ```
 
  If you use command prompt on Windows, you need to escape the quotes:
  
   ``` -d "{\"param\":\"value\"}" ```
 
 #### Avoiding Conflicts with Configuration on AirGradient Server
+
 If the monitor is set up on the AirGradient dashboard, it will also receive configurations from there. In case you do not want this, please set `configurationControl` to `local`. In case you set it to `cloud` and want to change it to `local`, you need to make a factory reset. 
```

Comment:
Propose to rephrase from

it will also receive configurations from there
to
it will also receive the configuration parameters from there